projects.md

```diff
@@ -78,6 +78,20 @@ send push notifications to any device, regardless of platform or network.
 
 <div class="project-title" markdown="1">
 
+## FHC
+
+FeedHenry Command Line Interface
```

Comment:
@camilamacedo86 I think it's fine to include a new category and the FHC repo under it. I think the category should be more general though - perhaps `FH Tooling`, so other repos other than FHC can be included in it and not just have a category with one repo within it

/my2cents

cc'ing @david-martin for his thoughts

Reply:
If there are other tools, sure.
If its just FHC, i don't think its that much a problem. But maybe the category could be 'CLI' then

Reply:
@damienomurchu @david-martin  I don't know any other project that we can add here in order to create a category CLI or FH Tooling. However, please let me know if you would like to change it anyway.